devbox.json

```diff
@@ -0,0 +1,12 @@
+{
+  "packages": [
```

Comment:
We should probably add a Java version here.

I just need to figure out which version of Java we're currently using...

Reply:
Actually, it looks like we might not need it.

I tried running Devbox with `--pure` and it worked fine as-is, so I suspect Gradle is pulling in Java automatically.